### DISCUSSION TITLE
Launch extension - black box + clear all my settings

### DISCUSSION THREAD

**Discussion by LightVolk** [coddingtonbear/obsidian-web#192] [Bug Report]
### Describe the bug

Suddenly, when i launch extension - i have a black box.
So, i go into settings - i lost my token and lost all of my templates.

So, i go to import settings and try to load saved temples . But nothing happend!

Also, i get token from Local RestAPI and put token into plugin.
But nothing happens.

I get this bug vey often , since last months.

### To Reproduce

![image](https://github.com/coddingtonbear/obsidian-web/assets/1236916/e87346a1-c6f9-4fe6-a223-52cbb310d1d2)

![image](https://github.com/coddingtonbear/obsidian-web/assets/1236916/95742e97-b43e-4fcb-9961-4076c60cbfae)

![image](https://github.com/coddingtonbear/obsidian-web/assets/1236916/536a2658-0296-49c1-ae2a-8db05b736d07)

![image](https://github.com/coddingtonbear/obsidian-web/assets/1236916/b406feb4-153d-4d42-8b60-d799ded72e57)

Bug report:

[obsidian-local-rest-api.bug_report (1).json](https://github.com/coddingtonbear/obsidian-web/files/15209911/obsidian-local-rest-api.bug_report.1.json)

I also add certificate, but it not work for me.

So, i often try to re-add my settings manually and it works some time.

I think that bug can be reproduced if open extension before launch Obsidian. 

But today this bug appeared when Obsidian launched before.

### Operating System

Windows 11

### Browser

Chrome

### Browser Version

124.0.6367.119 (64 bit)

**Comment by m0xsec** [comment]
running into this too!
